Shared/Extensions/WKWebViewExtensions.swift

```diff
@@ -0,0 +1,68 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import WebKit
+
+extension WKUserContentController {
+    public func addInDefaultContentWorld(scriptMessageHandler: WKScriptMessageHandler, name: String) {
+        if #available(iOS 14.0, *) {
+            add(scriptMessageHandler, contentWorld: .defaultClient, name: name)
+        } else {
+            add(scriptMessageHandler, name: name)
+        }
+    }
+}
+
+extension WKUserScript {
+    public class func createInDefaultContentWorld(source: String, injectionTime: WKUserScriptInjectionTime, forMainFrameOnly: Bool, sandboxed: Bool = true) -> WKUserScript {
+        if #available(iOS 14.0, *), sandboxed {
+            return WKUserScript(source: source, injectionTime: injectionTime, forMainFrameOnly: forMainFrameOnly, in: .defaultClient)
+        } else {
+            return WKUserScript(source: source, injectionTime: injectionTime, forMainFrameOnly: forMainFrameOnly)
+        }
+    }
+}
+
+public extension WKWebView {
+    private func generateJavascriptFunctionString(functionName: String, args: [Any]) -> String {
```

Comment:
would be good to have some test cases for this

Reply:
added: https://github.com/brave/brave-ios/pull/3048/files#diff-25e912adf660b51e9554179089070fe46ba13e52a5a974129cebeb614e327361